warehouse/templates/manage/account.html

```diff
@@ -713,6 +713,34 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
         {% trans href=request.route_path('organizations.profile', organization=event.additional.organization_name), organization_name=event.additional.organization_name, role_name=event.additional.role_name|lower %}Invitation to join <a href="{{ href }}">{{ organization_name }}</a> expired{% endtrans %}
       </strong>
 
+
+      {% elif event.tag == EventTag.Account.OrganizationRoleAdd %}
+      <strong>{% trans %}OrganizationRoleAdd{% endtrans %}</strong>
```

Comment:
I'm not sure what extra info to add, given inconsistent additional data, ref lines:

https://github.com/pypi/warehouse/blob/main/warehouse/organizations/services.py#L195-L214

Reply:
My mistake, these are different constants :-), deleting the comments!